state-transition/core/state_processor_genesis.go

```diff
@@ -179,7 +179,7 @@ func (sp *StateProcessor[
 			)
 		}
 		minEffectiveBalance := math.Gwei(
-			sp.cs.EjectionBalance() + sp.cs.EffectiveBalanceIncrement(),
+			sp.cs.EjectionBalance(false) + sp.cs.EffectiveBalanceIncrement(),
```

Comment:
@calbera question: should the Increment change too? I would say so.

Reply:
done